tools/BuildMonitor/BuildMonitor.csproj

```diff
@@ -51,7 +52,6 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DeployExtension>False</DeployExtension>
```

Comment:
Debug builds for BuildMonitor have already failed during restore. I'm not so sure this will fix this issue.